packages/compiler/src/server/server.ts

```diff
@@ -48,7 +49,19 @@ function main() {
   console.error = (data: any, ...args: any[]) => connection.console.error(format(data, ...args));
 
   const host: ServerHost = {
-    compilerHost: NodeHost,
+    compilerHost: {
+      ...NodeHost,
+      getJsImport: (path: string) => {
```

Comment:
only doing this in the server right now but feels like this probably should just be in the `NodeHost` because right now the repro will let tsp compile stuck